splinter/driver/webdriver/firefox.py

```diff
@@ -17,6 +17,11 @@
 from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
 from selenium.webdriver.firefox.options import Options
 
+try:
+    basestring
+except NameError:  # Python 3.x
+    basestring = str
```

Comment:
splinter uses `six` to helps with the python 2/3 compatibility. So you should use `six.string_types` (https://six.readthedocs.io/#six.string_types) instead to handle the `basestring` compatibility here.